### ISSUE TITLE
Starting application fails

### ISSUE THREAD

**Issue by mikepack** [dynamo/dynamo#28]
From within a fresh Dynamo master clone, I run the following commands:

``` bash
$ mix dynamo ../dynamo_app
$ cd ../dynamo_app
$ mix do deps.get, server
** (Mix) The task app.start could not be found
```

`mix help` doesn't list app.start, or the app namespace at all. Looks like `dynamo.start.ex` is [attempting to run it](https://github.com/josevalim/dynamo/blob/master/lib/mix/tasks/dynamo.start.ex#L17).

**Comment by josevalim** [comment]
Hey @mikepack. In order to run Dynamo, you need Elixir master, since it still is in full steam development. Please give a go on Elixir master and let us know.

How is Elixir treating you so far?

**Comment by mikepack** [comment]
Running off master fixed this and #27 for me. The confusion was that I thought `brew install --HEAD elixir` would install from master, but after looking at the formula, I can see [HEAD is actually stable](https://github.com/mxcl/homebrew/blob/master/Library/Formula/elixir.rb#L29). Might be breaking element of least surprise here, considering the Homebrew man states:

```
If --HEAD is passed, and formula defines it, install the HEAD version, aka master, trunk, unstable.
```

Either way, I got master installed via:

``` bash
$ make install
```

...which worked beautifully.

Elixir has been smooth so far. I'm enjoying the philosophies and choice of syntax. As a Rubyist, I like the notion that function bodies are wrapped in do..end, like procs as blocks. I'll be toying around with it more and putting together some trivial apps.
